### ISSUE TITLE
OS restarts

### ISSUE THREAD

**Issue by melvinotieno** [fhsinchy/tent#19]
Does using tent start <service> persist the podman container after OS restarts? If so, then that's not the case on my OS.

Fedora 34, running latest version of podman and tent.

**Comment by fhsinchy** [comment]
It depends on what you do before restarting your OS. If you stop the container by executing `tent stop mongo` then the container will be removed but the data volume will be kept. The next time you start the service a new container will be created.

If you restart the OS without executing `tent stop mongo` however, the container will be kept. So the next time you start the service the old container will be restarted.

The output of the `start` and `stop` commands actually prints out if the a new container is created or not or if a container is being removed.

**Comment by melvinotieno** [comment]
So if I don't do tent stop mongo before restarting, the container will not automatically start? So I have to start it again?

**Comment by fhsinchy** [comment]
You'll have to start the container everytime you boot your computer. No matter if yiu stoooed it or not. The services are not registered to start on startup. They're just containers giving an illusion of having a service running. I hope that makes it clearer mate.

**Comment by melvinotieno** [comment]
Yes, it does. Thank you for the clarification. Maybe as a future update you could add a flag to register it as a service that starts on start up if it doesn't complicate things.

**Comment by fhsinchy** [comment]
This was requested during the Podman Community meeting as well. I'll have a look into implementing this once I have some more free time for sure.

**Comment by melvinotieno** [comment]
Alright. Thank you for your time.

**Comment by melvinotieno** [comment]
Any progress on this?

**Comment by fhsinchy** [comment]
No not yet. I'm a little busy these days sadly. Also need to update the codebase with the latest tagged version of their bindings. Right now, I can't give any ETA either.

**Comment by fhsinchy** [comment]
Hey @melvinotieno I know I'm very late to the party but I just made a [commit](https://github.com/fhsinchy/tent/commit/644aa160bbb92705ef56d980db84ee8981b30e30) that adds the capability of keeping a container running even if the OS restarts.
